ogr/factory.py

```diff
@@ -97,11 +101,12 @@ def get_project(
                 service = service_inst
                 break
         else:
-            raise OgrException(
-                f"Instance of type {kls.__name__} "
-                f"matching instance url '{url}' was not provided."
-            )
-    else:
+            if force_custom_instance:
```

Comment:
could be merged into `elif` :)

Reply:
That's `for-else` and `for-elif` is not possible..;)

Reply:
oh my, there's `for` above it :D it didn't seem like that in diff :D